src/web/client/src/components/Register/Register.test.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FormEvent } from "react";
+import { shallow } from "enzyme";
+import { render } from "@testing-library/react";
+import { BrowserRouter as Router } from "react-router-dom";
+import Register from "./Register";
+
+describe("Register component", () => {
+  let registerWrapper: any;
+  let onAuthChangeMock: jest.Mock;
+  let historyMock: object;
+  let eventMock: any;
```

Comment:
eventmock is an object

Reply:
It gives me `Property 'target' does not exist on type 'object'.`